examples/channel-buffering/channel-buffering.go

```diff
@@ -11,17 +11,16 @@ import "fmt"
 
 func main() {
 
-    // Here we `make` a channel of strings buffering up to
-    // 2 values.
+    // Qui creiamo un channel (con `make`) di string con un 
+    // buffer di dimensione 2.
     messages := make(chan string, 2)
 
-    // Because this channel is buffered, we can send these
-    // values into the channel without a corresponding
-    // concurrent receive.
+    // Dato che questo channel è bufferizzato, possiamo inviare questi 
+    // due valori sul canale senza bloccarsi.
     messages <- "buffered"
     messages <- "channel"
 
-    // Later we can receive these two values as usual.
+    // Possiamo recuperare successivamente i valori lasciati sul canale.
```

Comment:
Cambia canale con channel, poi io in #45 ho tradotto send come inviare e receive come ricevere, mentre qui posso vedere che hai usato "lasciati". Cambio io o cambi tu?